tests/test_portable.py

```diff
@@ -0,0 +1,30 @@
+from os import path
+import sys
+import subprocess
+import dash
+
+
+def test_portable(dash_duo, tmpdir):
+    # Build a portable webviz from config file
+    appdir = path.join(tmpdir, 'app')
+    subprocess.call(['webviz', 'build', 'basic_example.yaml',
+                     '--portable', appdir], cwd='examples')
+    # Remove Talisman
+    fn = path.join(appdir, 'webviz_app.py')
+    with open(fn, "r") as f:
+        lines = f.readlines()
+    with open(fn, "w") as f:
+        for line in lines:
+            if not line.strip("\n").startswith("Talisman"):
+                f.write(line)
```

Comment:
Is it possible to instead change selenium to allow untrusted/self-signed certificates?
https://stackoverflow.com/posts/25063706/revisions